snow/consensus/snowman/consensus_test.go

```diff
@@ -1082,7 +1082,7 @@ func RecordPollDivergedVotingWithNoConflictingBitTest(t *testing.T, factory Fact
 	votes3 := bag.Of(block3.ID())
 	require.NoError(sm.RecordPoll(context.Background(), votes3))
 
-	require.False(sm.Finalized(), "finalized too early")
+	require.NotZero(sm.NumProcessing())
```

Comment:
Seems a bit strange to use NotZero versus checking the actual num in a unit test

I'm fine either way since we were doing it this way before just through the Finalized helper